.github/workflows/llm_bench-python.yml

```diff
@@ -74,6 +74,16 @@ jobs:
       run: |
         optimum-cli export openvino --model segmind/tiny-sd --trust-remote-code --weight-format fp16 ./ov_models/tiny-sd/pytorch/dldt/FP16/
         python ./tools/llm_bench/benchmark.py -m ./ov_models/tiny-sd/pytorch/dldt/FP16/ -pf ./tools/llm_bench/prompts/stable-diffusion.jsonl -d cpu -n 1
+    - name: Test whisper-tiny on Linux
+      run: |
+        GIT_LFS_SKIP_SMUDGE=1 git clone --depth 1 --branch main --single-branch https://huggingface.co/datasets/facebook/multilingual_librispeech
+        cd multilingual_librispeech
+        git lfs pull -I /data/mls_polish/train/audio/3283_1447_000.tar.gz
+        mkdir data/mls_polish/train/audio/3283_1447_000
+        tar zxvf data/mls_polish/train/audio/3283_1447_000.tar.gz -C data/mls_polish/train/audio/3283_1447_000/
+        cd ..
+        optimum-cli export openvino --trust-remote-code --model openai/whisper-base ./ov_models/whisper-base
```

Comment:
maybe we can use smaller size model, what about whisper-small or whisper-tiny?

Reply:
using whisper-tiny.